_includes/v21.1/orchestration/start-cockroachdb-helm-secure.md

```diff
@@ -1,5 +1,9 @@
 {{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is undergoing maintenance for compatibility with Kubernetes versions 1.17 through 1.21 (the latest version as of this writing). No new feature development is currently planned. For new production and local deployments, we currently recommend using a manual configuration (**Configs** option). If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.17 through 1.22 (the latest version as of this writing). No new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_danger}}
+If you are already running a secure Helm deployment on Kubernetes 1.22 and later, you must migrate away from using the Kubernetes CA for cluster authentication. For details, see [Operate CockroachDB on Kubernetes](operate-cockroachdb-kubernetes.html?filters=helm#migration-to-self-signer).
```

Comment:
Not sure why this is a separate warning. Feels like it should be combined with the one above.